tools/integration-test/src/tests/client_expiration.rs

```diff
@@ -298,55 +309,14 @@ impl BinaryChainTest for PacketExpirationTest {
                 &denom_a,
             )?;
 
-            {
-                info!("sending first IBC transfer after client is expired. this should cause packet worker to fail");
-
-                chains.node_a.chain_driver().transfer_token(
```

Comment:
The tests here has to be removed, because since Gaia v7 the transaction will fail. Apparently we didn't catch this error previously, because submitting the transaction via the `gaiad` command do not result in error.